ui/components/multichain-accounts/add-wallet-modal/add-wallet-modal.tsx

```diff
@@ -25,6 +26,11 @@ import {
 import type { ModalProps } from '../../component-library';
 import { FlexDirection } from '../../../helpers/constants/design-system';
 import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  CONNECT_HARDWARE_ROUTE,
+  IMPORT_SRP_ROUTE,
+  NEW_ACCOUNT_ROUTE,
```

Comment:
Change this to `ADD_WALLET_PAGE_ROUTE` once new Add wallet page for private key import has been merged. https://github.com/MetaMask/metamask-extension/pull/35543

Reply:
updated https://github.com/MetaMask/metamask-extension/pull/35536/commits/fd6911761a241d54cf8e5898752a0c8e49d1d874